skimage/transform/radon_transform.py

```diff
@@ -171,8 +172,9 @@ def _get_fourier_filter(size, filter_name):
     return fourier_filter[:, np.newaxis]
 
 
+@deprecate_arg({'filter': 'filter_name'})
 def iradon(radon_image, theta=None, output_size=None,
-           filter="ramp", interpolation="linear", circle=True):
+           filter_name="ramp", interpolation="linear", circle=True):
```

Comment:
Generally you need to add warnings so the old behavior can be taken out.

Reply:
I am sorry, but don't understand what you mean here :-S. The warning is raised by the decorator.

Reply:
You are right 0!